inc/shortcodes/class-iframe.php

```diff
@@ -50,9 +50,11 @@ public static function callback( $attrs, $content = '' ) {
 		}
 
 		return sprintf(
-			'<iframe src="%s" width="%s" height="%s" frameborder="0" scrolling="no" class="shortcake-bakery-responsive"></iframe>',
+			'<iframe src="%s" width="%s" height="%s" data-true-width="%s" data-true-height="%s" frameborder="0" scrolling="no" class="shortcake-bakery-responsive"></iframe>',
```

Comment:
Oh, this is how the responsiveness JS knows which proportion to maintain. I guess we were missing it before?